packages/cli-kit/src/node/ruby.ts

```diff
@@ -44,11 +44,16 @@ export async function execCLI2(args: string[], {adminSession, storefrontToken, d
     BUNDLE_GEMFILE: join(shopifyCLIDirectory(), 'Gemfile'),
   }
 
-  await system.exec('bundle', ['exec', 'shopify'].concat(args), {
-    stdio: 'inherit',
-    cwd: directory ?? process.cwd(),
-    env,
-  })
+  try {
+    await system.exec('bundle', ['exec', 'shopify'].concat(args), {
+      stdio: 'inherit',
+      cwd: directory ?? process.cwd(),
+      env,
+    })
+  } catch (error) {
+    // CLI2 will show it's own errors, we don't need to show an additional CLI3 error
+    throw new AbortSilent()
```

Comment:
That's a good question 🤔 
Maybe CLI3 shouldn't report CLI2 issues to bugsnag (because it will just be a CLI2 failed with details)
CLI2 already reports issues to its own bugsnag, make sense to keep it separate?

Reply:
Just double checked. AbortSilent won't report errors to Bugsnag, and I think that's ok.
CLI2 will report the error if needed.

Idea: We should add some variable in the bug report from CLI2 to identify that it was executed from CLI3

Reply:
We can use the new variable I've added to filter warnings, once we've decided on a name.